From e337d28d48359f149ff68a5bc2b77bf87675d7bc Mon Sep 17 00:00:00 2001
From: "K.Takata" <kentkt@csc.jp>
Date: Tue, 2 Oct 2018 15:18:16 +0900
Subject: [PATCH 13/28] nsis: Add workarounds for Japanese NSIS messages

Some messages are too long and not shown properly.
Also the access key was missing in a message.
These should be fixed by the NSIS upstream, I think.
---
 nsis/lang/japanese.nsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/nsis/lang/japanese.nsi b/nsis/lang/japanese.nsi
index 93cae55ce..cc6282143 100644
--- a/nsis/lang/japanese.nsi
+++ b/nsis/lang/japanese.nsi
@@ -18,6 +18,16 @@ LangString ^SetupCaption     ${LANG_JAPANESE} \
 LangString ^UninstallCaption ${LANG_JAPANESE} \
         "$(^Name) Uninstall"
 
+# Workarounds for NSIS Japanese translation. The messages are too long.
+# These should be better to be fixed by the NSIS upstream.
+LangString ^SpaceAvailable   ${LANG_JAPANESE} \
+        "利用可能なディスク容量："
+LangString ^SpaceRequired    ${LANG_JAPANESE} \
+        "必要なディスク容量："
+# Fix another NSIS Japanese translation. The access key was missing.
+LangString ^InstallBtn       ${LANG_JAPANESE} \
+        "インストール(&I)"
+
 ##############################################################################
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
-- 
2.17.0

